### DISCUSSION TITLE
Is there a way to produce a tagged PDF using Quarto?

### DISCUSSION THREAD

**Discussion by ABSOD** [quarto-dev/quarto-cli#9333] [Q&A]
### Description

I need to produce multiple accessible PDF documents (500+ documents) using an existing R code base.   I have seen where R Markdown supports alternative text specifications (https://posit.co/blog/knitr-fig-alt/), but my experiments suggest that this alternative text is not included in the PDF output.  The documents I need to produce have to pass WCAG standards. 

I've read about the issues with automatic tagging, but I'm trying to avoid manual tagging wherever possible.

Can Quarto produce a tagged PDF?  The final document should support a defined reading order, alternative text for images, and a layout for tables.  What kind of LaTEX/Pandoc/Quarto specifications produce an accessible document?
TIA

**Comment by jimjam-slam** [comment]
I feel like this is something that [Pandoc has struggled with](https://github.com/jgm/pandoc/issues/5409), and I don't think its currently proposed options for making tagged PDFs are great. I'm crossing my fingers that the [Typst team will get there soon](https://github.com/typst/typst/issues/133), though!

**Comment by tomhanna-uh** [comment]
This suggestion was posted more than 2 years ahead of the federal government deadline on this. We are now down to less than 3 months until all PDFs have to be tagged. This is to the point of being urgent. bump bump bump

**Reply by cderv** [reply]
I think there is other ways to ask for feature than "bump bump bump". Please be respectful of us developer and contributor to this project. This is open source and we are always doing the best we can in provided time constraint and many many projects to work on. 

That being said about the form of your comment, please when you look for something, do search in the repo, especially for any related updates. You would have found the following: 

- https://github.com/quarto-dev/quarto-cli/discussions/13746 previous discussion
- https://github.com/quarto-dev/quarto-cli/issues/4426 previous issue now closed 
- https://github.com/quarto-dev/quarto-cli/pull/13857 PR implement some new feature

So this is in the work for 1.9 and having search carefully you would have had hint on that. 

So please next time, do a bit more than commenting on an old issue asking us for update. Being part of a community in open source is a two way system: Maintainers always doing there best with their constraints, and community members being helpful and respectful on the most valuable ressource of the developer: Time. 

Hope you understand. Thank you. 

And you should be able to try soon in next 1.9 pre release the new `pdf-standard` flag. There is already some documentation on prerelease doc
- https://prerelease.quarto.org/docs/output-formats/pdf-basics.html#pdf-accessibility-standards
- https://prerelease.quarto.org/docs/output-formats/typst.html#pdf-accessibility-standards
